crates/wash-cli/tests/wash_link.rs

```diff
@@ -38,3 +38,36 @@ async fn integration_link_serial() -> Result<()> {
 
     Ok(())
 }
+
+/// Ensure wash can delete all links
+#[tokio::test]
+#[serial]
+async fn integration_link_del_all_serial() -> Result<()> {
```

Comment:
I think it would be good for this test to define a few different links and then ensure that the delete all deletes all of them